GrocifyApp.UI/src/consts/ENConsts.js

```diff
@@ -67,6 +68,7 @@ const PassRulesConsts = {
 
 const ModalConsts = {
     DefaultShoppingList: "Change default shopping list",
+    DeleteSection: (section) => `Are you sure you want to delete "${section}" section?`,
```

Comment:
I think "Are you sure you want to delete ${something}" should be generic.

And here title={ModalConsts.DeleteSection(selectedSection.name + " section")};

you don't need to make section as a constant because i already made it in my PR.  Just add a todo for later update